09-Palindrome_Number.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+'''
+Determine whether an integer is a palindrome. An integer is a palindrome when it
+reads the same backward as forward.
+
+Example 1:
+
+Input: 121
+Output: true
+Example 2:
+
+Input: -121
+Output: false
+Explanation: From left to right, it reads -121. From right to left, it becomes
+121-. Therefore it is not a palindrome.
+Example 3:
+
+Input: 10
+Output: false
+Explanation: Reads 01 from right to left. Therefore it is not a palindrome.
+Follow up:
+
+Coud you solve it without converting the integer to a string?
+'''
+
+class Solution:
+    def isPalindrome(self, x):
+        """
+        :type x: int
+        :rtype: bool
+        """
+        return str(x) == str(x)[::-1]
+
+# Solution without converting the integer to a string
+class Solution2:
+    def isPalindrome(self, x):
+        """
+        :type x: int
+        :rtype: bool
+        """
+        reversedNum = 0
```

Comment:
There is a code style guidance to declare a variable close to where you use it.
In this case, you should move the declarations before your loop.
There's no need to declare these if x < 0.

Reply:
done